### DISCUSSION TITLE
Bevy WASM with trunk error loading gltf

### DISCUSSION THREAD

**Discussion by Caligino** [bevyengine/bevy#5972] [Q&A]
OK I managed to get my app to compile with trunk but when i go to serve it for testing it fails to load the glb asset and presumably it crashes. I added the copy options to the trunk config html file but still get the same error:
```WARN C:\Users\Aster\.cargo\registry\src\github.com-1ecc6299db9ec823\bevy_asset-0.8.1\src\asset_server.rs:455 encountered an error while loading an asset: invalid glTF file: expected value at line 1 column 1```
My only insight is that it might actually only want to load a gltf and not a glb file?
Does anyone, preferably a wizard have any clue whats going on?

**Comment by mockersf** [comment]
A first guess would be that `trunk` is not serving asset files properly. Could you check in your browser what is the response to the http request for the asset?
With a quick look at their doc, I'm guessing you must use `rel="copy-dir"` with the assets directory

**Reply by Caligino** [reply]
i did this. 
```
<html>
  <head>
    <link data-trunk rel="css" href="css/style.css"/>
    <link data-trunk rel="copy-dir" href="assets"/>
    <link data-cargo-all-features rel="rust" href="Cargo.toml"/>
    <link data-wasm-opt="s"/>
    <link data-type rel="rust" href="main.rs"/>
  </head>
</html>
```

still no idea what the heck. the out put wasm doesnt appear to be optimized although both the html and cargo.toml configs instruct the compiler to.
